lib/internal/util/parse_args/parse_args.js

```diff
@@ -44,6 +44,7 @@ const {
     ERR_PARSE_ARGS_INVALID_OPTION_VALUE,
     ERR_PARSE_ARGS_UNKNOWN_OPTION,
     ERR_PARSE_ARGS_UNEXPECTED_POSITIONAL,
+    ERR_PARSE_ARGS_REQUIRED_OPTION
```

Comment:
Let's use trailing commas where we can
```suggestion
    ERR_PARSE_ARGS_REQUIRED_OPTION,
```